### DISCUSSION TITLE
Disable to delete some intellisenses

### DISCUSSION THREAD

**Discussion by supersovaa** [James-Yu/LaTeX-Workshop#4137] [Q&A] [answered]
I want to delete some intellisenses which I never use.
Some intellisenses could be deleted, but other intellisenses like "labelformat", "labelitemfont" could not.
How can I delete them?
![image](https://github.com/James-Yu/LaTeX-Workshop/assets/86138538/acbc58eb-5a9d-4d45-8932-130094a1eb77)
![image](https://github.com/James-Yu/LaTeX-Workshop/assets/86138538/85ecf13d-49a1-4518-bcf5-e8706408b1f5)

```
"latex-workshop.intellisense.command.user": {
        "labelformat": "",
        "labelitemfont": "",
    },
    "latex-workshop.intellisense.package.exclude": [
        "latex-document",
    ],
'''

**Comment by jlelong** [comment, marked as answer]
You may try to exclude the package `lw-default` to remove all default commands and environments. See https://github.com/James-Yu/LaTeX-Workshop/wiki/Intellisense#latex-workshopintellisensepackageexclude.

The macro `\labelformat{}{}` is provided by the pseudo package `latex-document.json`. No macro `\labelformat` (without the arguments) is defined.  So you can safely remove it from `"latex-workshop.intellisense.command.user"`.

Note that you can only remove commands defined `data/commands.json` by setting their definition as void in `"latex-workshop.intellisense.command.user"`. So it will not work with `\labelformat{}{}`. There is no way to remove commands defined in a package one by one.
